h2o-genmodel/src/main/java/hex/genmodel/easy/EnumLimitedEncoderColumnMapper.java

```diff
@@ -0,0 +1,25 @@
+package hex.genmodel.easy;
+
+import hex.genmodel.GenModel;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class EnumLimitedEncoderColumnMapper {
```

Comment:
this looks exactly the same as EnumEncoderColumnMapper we should not just copy-paste code

Reply:
there is a difference in line 17: EnumLimitedEncoderColumnMapper takes _m.getOrigNames() as modelColumnNames and EnumEncoderColumnMapper takes _m.getNames() as modelColumnNames